package.json

```diff
@@ -47,7 +47,7 @@
     "@ngrx/effects": "^8.2.0",
     "@ngrx/entity": "^8.2.0",
     "@ngrx/store": "^8.2.0",
-    "@ngu/carousel": "^1.5.5",
```

Comment:
we have now support for Angular 9 with SSR , try to use 2.0.3.

Reply:
Thanks for pointing that out. Will update that.
I'm just working on fixing universal build process.